### PR TITLE
Pandoc 3.1.11.1

### PR DESCRIPTION
**Pull request by SaravananSathyanandhaQC**
WIP (and first time contributing here so could likely use some help with testing)

- As per comments in https://github.com/conda-forge/pandoc-feedstock/pull/138 this is a proposal to switch to repackaging for all binaries, as getting the build to work with `stack` is proving to be difficult
- Bumps to 3.1.11.1 (could alternatively go lower then keep bumping to get some intermediate versions released too)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
LGTM! Thanks!!

@m-rossi what do you think? I'd love a second pairs of eyes here before we get back to the pre-packaged pandoc.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,15 +25,8 @@ build:
...
-  build:
-    - pkg-config  # [osx and not arm64]
-    - stack  # [not aarch64 and not arm64]
   host:
-    - zlib  # [linux and not aarch64]
-    - zlib  # [osx and not arm64]
-  run:
-    - gmp  # [linux and not aarch64]
-    - zlib  # [linux and not aarch64]
+    - zlib
```
Comment (m-rossi) [inline recipe/meta.yaml:29]: I don't think we need `zlib` in general on host and it was not required before I tried to build it on source.

**Review by m-rossi — CHANGES_REQUESTED** [review]
@SaravananSathyanandhaQC thanks for updating the feedstock. Looks mostly good for me. Just one thing.

File: recipe/meta.yaml (hunk 1 above)
Comment (SaravananSathyanandhaQC) [reply recipe/meta.yaml]: Good spot, thanks!

**Comment by m-rossi** [comment]
Could be we need a regender here, let's be safe

@conda-forge-admin, please rerender

**Review by m-rossi — APPROVED** [review]

**Comment by m-rossi** [comment]
Thanks @SaravananSathyanandhaQC, merging now.

**Pull request merged**